src/pushNotifications/widget/pushNotifications.js

```
@@ -307,6 +336,49 @@ define([
 
         removeAlert: function (e){
             e.parentNode.parentNode.removeChild(e.parentNode);
+        },
+
+        _executeOfflineOnline: function (getGCMSettingsEntityOfflineFn, getGCMSettingsEntityOnlineFn) {
+            if (this.version.major > 7 || (this.version.major === 7 && this.version.minor >= 3)) {
```

I think it is better to throw an exception when major version exceeds seven